_posts/multirepo_comments.adoc

```diff
@@ -0,0 +1,96 @@
+# The Dangers of Multirepo
+
+In this post I'm going to make the case for a somewhat unpopular opinion - using multiple git repos (AKA multirepo) is unnecessary and dangerous in many cases. A single git repo (AKA monorepo) is simpler and satisfies the requirements of many projects.
```

Comment:
You're attacking the subject without giving some background on the problem. Try starting with a story maybe?

"At my last company we had a project yada yada yada"